amplify/backend/backend-config.json

```diff
@@ -52,6 +52,11 @@
       "build": true,
       "providerPlugin": "awscloudformation",
       "service": "Lambda"
+    },
+    "getUserReputation": {
+      "build": true,
+      "providerPlugin": "awscloudformation",
+      "service": "Lambda"
```

Comment:
I'm surprised to see this here, considering it's already on lines 41-45. Can you confirm this is intentional?

Reply:
Looks like I had local changes for this file that were not pushed. Should be fixed now.